### ISSUE TITLE
Write the "Modules" concept

### ISSUE THREAD

**Issue by 0xNeshi** [exercism/cairo#38]
Official docs: 
https://book.cairo-lang.org/ch07-00-managing-cairo-projects-with-packages-crates-and-modules.html

The idea is to have a short and to-the-point article about the concept. Avoid copy/pasting everything from The Cairo Book; the Book should be used as a more in-depth learning resource for students after they finish the concept on Exercism (or they can use it in parallel). 

This should focus only on the modules, as this can be practiced on Exercism. Packages and crates are out of scope, and students will have to dive into those on their own.

---
Exercism Concepts docs:
https://exercism.org/docs/building/tracks/concepts

Use the following Rust's concept as inspiration for the format:
https://github.com/exercism/rust/blob/main/concepts/structs/about.md
